web-console/src/lib/components/connectors/dialogs/KafkaInputConnector.tsx

```diff
@@ -20,9 +20,13 @@ import { useEffect, useState } from 'react'
 import { FieldErrors } from 'react-hook-form'
 import { FormContainer } from 'react-hook-form-mui'
 import * as va from 'valibot'
+import IconCategoryAlt from '~icons/bx/category-alt'
+import IconData from '~icons/bx/data'
+import IconFile from '~icons/bx/file'
+import IconLockOpen from '~icons/bx/lock-open'
+import IconX from '~icons/bx/x'
```

Comment:
what does the ~ do

Reply:
It's just a convention of 'unplugin-icons' to generate module names =/